fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/common/organisation/CampaignsHelper.java

```diff
@@ -53,50 +53,50 @@ public CampaignsHelper(final RequestSpecification requestSpec, final ResponseSpe
 
     public Integer createCampaign(String reportName, Integer triggerType) {
         LOG.info("---------------------------------CREATING A CAMPAIGN---------------------------------------------");
-        final String CREATE_SMS_CAMPAIGNS_URL = SMS_CAMPAIGNS_URL + "?" + Utils.TENANT_IDENTIFIER;
-        return Utils.performServerPost(requestSpec, responseSpec, CREATE_SMS_CAMPAIGNS_URL, getCreateCampaignJSON(reportName, triggerType),
+        final String createSMScampaignsURL = SMS_CAMPAIGNS_URL + "?" + Utils.TENANT_IDENTIFIER;
```

Comment:
We've made SMS lowercase elsewhere, so would these make more sense as createSmsCampaignsURL etc?

Reply:
updated.